components/tooltip/index.css

```diff
@@ -55,6 +55,10 @@ governing permissions and limitations under the License.
   /* tip */
   --spectrum-tooltip-tip-inline-size: var(--spectrum-tooltip-tip-width);
   --spectrum-tooltip-tip-block-size: var(--spectrum-tooltip-tip-height);
+  /* Percentage value of tip width divided by tip height, for calculating clip-path within a square tip. */
+  --spectrum-tooltip-tip-height-percentage: 50%;
```

Comment:
Note: I wish I could calculate this with a CSS calc() from `--spectrum-tooltip-tip-width` and `--spectrum-tooltip-tip-height`, but it's not possible to divide with both tokens having px units.